fixtures/small/binary_operators_expected.rb

```diff
@@ -1,6 +1,7 @@
 foo || bar
 
-foo || bar
+foo ||
+  bar
```

Comment:
For basically all of these fixture updates, in the original versions, these were multilined, so keeping them multiline seems correct (and consistent with how we handle every other breakable)